packages/web3torrent/package.json

```diff
@@ -131,6 +131,7 @@
     "prettier:check": "prettier --check '**/*.{scss,json}' --ignore-path '.eslintignore'",
     "prettier:write": "prettier --write '**/*.{scss,json}' --ignore-path '.eslintignore'",
     "start": "node scripts/start.js",
+    "start:tracker": "node tracker/index.js",
```

Comment:
It doesn't currently seem possible to run this script if I already ran `yarn start` (a tracker is already running on that port). 

That means this command can only allow me to run a local tracker on its own, without the web3torrent client, which seems of limited use.

Reply:
It accepts the TRACKER_PORT environment variable. That allows for multiple trackers to be run at the same time in the same computer.

Reply:
By the way, why would you want to run it independently? From this PR onwards we would run it as part of the `yarn start` script.

Reply:
![image](https://user-images.githubusercontent.com/10502605/77305249-77620300-6cd4-11ea-8602-8dfb3b9ecaa5.png)
That way, we don't have to run yet another terminal

Reply:
Yeah we probably wouldn't need to run it separately; I just spotted the `yarn start:tracker` script and wondered what that was for :-) . I tried it out because I thought I might be able to see the logs that way.